code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -89,8 +89,11 @@
 			emote("me", EMOTE_AUDIBLE, "snuffles")
 
 	if(nutrition >= MOUSETFAT)
-		visible_message("<span class = 'warning'>\The [src] explodes!</span>")
-		gib()
+		if(!maxHealth == 35) //Only mouser rats have 35 max health. Technically there's a very small chance a regular rat gets adminbussed to 35max health, but the code has sanity checks so the transmog death doesn't runtime
```

Comment:
Shouldn't this be
`if(maxHealth != 35)`?

Reply:
Also this will break if anyone changes mouser mouse maxHealth in the future. Maybe use a define if you want to do it this way, but maybe a more explicit way to check that it's a mouser mouse would be  best.

Reply:
This would be one of the rare good uses of `initial()`.

Reply:
I first considered using a "ismouservictim" var but decided against it. wew